_d/osx.md

```
@@ -13,8 +13,16 @@ I pretty much use mac exclusively. I have 3 at work, and a mac mini at home, and
 <!-- prettier-ignore-start -->
 <!-- vim-markdown-toc GFM -->
 
-- [Alfred](#alfred)
-- [Random](#random)
+- [Apps](#apps)
+  - [Alfred - The everything launcher/doer!](#alfred-the-everything-launcher-doer)
+    - [3rd Party Plugins:](#3rd-party-plugins)
+    - [2nd Party - E.g., Igor wrote it.](#2nd-party-e-g-igor-wrote-it)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)